Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -1064,6 +1064,10 @@ MonoBehaviour:
   VObPerFrame: 75
   moveSpeedPlayerPref: MoveSpeed
   turnSettingPlayerPref: TurnSetting
+<<<<<<< HEAD
```

Comment:
Merge issues. Please solve. From what I see, I think the first one is right. The other doesn't contain any value.